pkg/segment/structs/evaluationstructs.go

```diff
@@ -120,6 +138,13 @@ const (
 	BoolOpOr
 )
 
+type StatisticExprMode uint8
+
+const (
+	SEMTop = iota
+	SEMRare
+)
```

Comment:
This `SEM...` naming convention is going to make it hard to know if its for a `StatisticExprMode` or for a `StringExprMode`, so you should change one or both of these so the modes for string and statistic expressions doesn't both follow the pattern `SEM...`